src/util/system.h

```diff
@@ -268,6 +268,22 @@ class ArgsManager
      */
     const fs::path& GetBlocksDirPath() const;
 
+    /**
+     * Get data directory path
+     *
+     * @return Absolute path on success, otherwise an empty path when a non-directory path would be returned
+     * @post Returned directory path is created unless it is empty
+     */
+    const fs::path& GetDataDirBase() const { return GetDataDirPath(false); }
+
+    /**
+     * Get data directory path with appended network identifier
+     *
+     * @return Absolute path on success, otherwise an empty path when a non-directory path would be returned
+     * @post Returned directory path is created unless it is empty
+     */
+    const fs::path& GetDataDirNet() const { return GetDataDirPath(true); }
+    
```

Comment:
d5f1458ea3821cdf87954f012bd1606659cf818f: This is adding trailing whitespace. Also the commit title is wrong. `Path` should be removed.